.azure-pipelines/nightly-CI.yml

```diff
@@ -91,6 +87,10 @@ jobs:
         python.version: '3.8'
         ONNX_PATH: onnx==1.8.0
 
+      Python39:
+        python.version: '3.9'
+        ONNX_PATH: onnx==1.9.0
```

Comment:
why not added the latest ones?

Reply:
I saw other pipeline, e.g. SKLEARN-ONNX, win32-CI-conda-CI.yml, has same settings.

Reply:
I tried to add python 3.10 and onnx=1.10.0, but the test failed.  let me try the python 3.9 and onnx 1.10.1 combination.